xbmc/addons/interfaces/General.cpp

```diff
@@ -55,10 +55,10 @@ void Interface_General::Init(AddonGlobalInterface* addonInterface)
   addonInterface->toKodi->kodi->get_region = get_region;
   addonInterface->toKodi->kodi->get_free_mem = get_free_mem;
   addonInterface->toKodi->kodi->get_global_idle_time = get_global_idle_time;
+  addonInterface->toKodi->kodi->kodi_version = kodi_version;
```

Comment:
Is moving this line necessary? If it is, this is the wrong place.

Reply:
Not direct necessary but to have for better overview in same flow as https://github.com/xbmc/xbmc/blob/master/xbmc/addons/kodi-addon-dev-kit/include/kodi/General.h#L47-L64

Reply:
> Not direct necessary

 Can we then please change this later, with the next change where ABI breakage is really unavoidable?

Reply:
order has to be then:
* get_global_idle_time
* get_current_skin_id
* kodi_version
* change_keyboard_layout
* get_keyboard_layout

Reply:
deleted my previous comments... was the wrong place.